File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int x = 0;
        for (auto operation : operations ) {
            string var = operation;
            if (var == "X++" || var == "++X") {
                x++;
            } else {
                x--;
            }
        }
        return x;
    }
};

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        op_dict = {"--X" : -1, "X--" : -1,
                 "++X" : 1, "X++" : 1}
        return sum(op_dict[op] for op in operations)